File: 06. Perulangan/G. Statistika Sederhana/g_statistika_sederhana.pas
program g_statistika_sederhana;
var
  N, i : integer;
  num, A, B: longint;
begin
  readln(N);
  for i := 1 to N do begin
    read(num);
    if (i = 1) then begin
      A := num;
      B := num;
    end
    else begin
      if (num > A) then A := num;
      if (num < B) then B := num;
    end;
  end;
  writeln(A, ' ', B);
end.

File: 06. Perulangan/G. Statistika Sederhana/g_statistika_sederhana.c
#include <stdio.h>
int main(){
  int N, i, num, A, B;
  scanf("%d", &N);
  for (i = 0; i < N; i++){
    scanf("%d", &num);
    if (i == 0){
      A = num;
      B = num;
    }
    else {
      if (num > A)
        A = num;
      if (num < B)
        B = num;
    }
  }
  printf("%d %d\n", A, B);
  return 0;
}
